pkgs/top-level/all-packages.nix

```diff
@@ -20380,7 +20380,7 @@ in
     geoip = geoipWithDatabase;
   };
 
-  notion = callPackage ../applications/window-managers/notion { };
+  notion = callPackage ../applications/window-managers/notion { lua = lua5_3; };
```

Comment:
Was this necessary? Notion should work fine with the current Nix default (5.2), I've been testing with that for months.

Reply:
Oh, it was a leftover of various tests.